### DISCUSSION TITLE
Button Icons broken

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#10515] [General - Components / Directives / etc] [answered]
I noticed when setting an button to be flat the icon disappears. has anyone else exp this? I'm on the latest version of quasar.

        `<q-btn color="primary" icon="mdi-arrow-left" dense flat/>`

**Comment by pikil** [comment, marked as answer]
@JokerMartini could you check if 'mdi-v5' is enabled in quasar.conf->extras section?

extras: [
    'mdi-v5',
    // 'material-icons', // optional, you are not bound to it
    ...
  ]
